spec/order_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
```

Comment:
This file has been moved to another directory in the meantime, so please delete it. :)